data/src/main/scala/ch.epfl.scala.index.data/bintray/BintrayMeta.scala

```diff
@@ -32,13 +30,4 @@ object BintrayMeta extends BintrayProtocol {
       .sortBy(_.created)(Descending)
   }
 
-  def append(paths: DataPaths, meta: BintraySearch): Unit = {
```

Comment:
It seems that it’s been superseeded by `writeMergedPoms` [here](https://github.com/scalacenter/scaladex/blob/master/data/src/main/scala/ch.epfl.scala.index.data/bintray/BintrayListPoms.scala#L132-L141).